libvast/vast/system/type_registry.hpp

```diff
@@ -50,7 +51,7 @@ using type_registry_type = caf::typed_actor<
   caf::replies_to<atom::get>::with<type_set>,
   caf::replies_to<atom::get, std::string>::with<type_set>,
   caf::reacts_to<atom::put, vast::taxonomies>,
```

Comment:
As discussed, I think it would be great if we could make `taxonomies` an implementation detail and not part of the public API.

```suggestion
  caf::reacts_to<atom::put, taxonomies>,
```

Reply:
As discussed in sync, we currently need this until the loading mechanism is in place.